ansible_collections/arista/avd/examples/cv-pathfinder/intended/configs/inet-cloud.cfg

```diff
@@ -0,0 +1,151 @@
+   no shutdown
+!
+dhcp server
+   lease time ipv4 6 days 23 hours 59 minutes
+   !
+   subnet 100.64.10.0/24
+      !
+      range 100.64.10.2 100.64.10.2
+      name 10 NET
+      default-gateway 100.64.10.1
+   !
+   subnet 100.64.11.0/24
+      !
+      range 100.64.11.2 100.64.11.2
+      name 11 NET
+      default-gateway 100.64.11.1
+   !
+   subnet 100.64.30.0/24
+      !
+      range 100.64.30.2 100.64.30.2
+      name 30 NET
+      default-gateway 100.64.30.1
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname inet-cloud
+ip name-server vrf MGMT 192.168.17.1
+dns domain wan.example.local
+!
+spanning-tree mode none
+!
+vrf instance MGMT
+!
+management api http-commands
+   protocol https
+   no shutdown
+   !
+   vrf MGMT
+      no shutdown
+!
+aaa authorization exec default local
+!
+interface Ethernet1
+   description pf1-Ethernet2
+   no shutdown
+   no switchport
+   ip address 100.64.100.1/24
+!
+interface Ethernet2
+   description pf2-Ethernet2
+   no shutdown
+   no switchport
+   ip address 100.64.200.1/24
+!
+interface Ethernet5
+   description site1-wan1-Ethernet4
+   no shutdown
+   no switchport
+   ip address 100.64.10.1/24
+   dhcp server ipv4
+!
+interface Ethernet6
+   description site1-wan2-Ethernet4
+   no shutdown
+   no switchport
+   ip address 100.64.11.1/24
+   dhcp server ipv4
+!
+interface Ethernet7
+   description site2-wan2-Ethernet4
+   no shutdown
+   no switchport
+   ip address 100.64.21.1/24
+!
+interface Ethernet8
+   description site3-wan1-Ethernet4
+   no shutdown
+   no switchport
+   ip address 100.64.30.1/24
+   dhcp server ipv4
+!
+interface Loopback0
+   description ROUTER_ID
+   no shutdown
+   ip address 172.31.255.23/32
+!
+interface Management1
+   description OOB_MANAGEMENT
+   no shutdown
+   vrf MGMT
+   ip address 192.168.17.31/24
+   no lldp transmit
+   no lldp receive
+!
+ip routing
+no ip routing vrf MGMT
+!
+ip prefix-list PL-LOOPBACKS-EVPN-OVERLAY
+   seq 10 permit 172.31.255.0/24 eq 32
+!
+ip route vrf MGMT 0.0.0.0/0 192.168.17.1
+!
+ntp local-interface vrf MGMT Management1
+ntp server vrf MGMT 0.pool.ntp.org prefer
+!
+route-map RM-CONN-2-BGP permit 10
+   match ip address prefix-list PL-LOOPBACKS-EVPN-OVERLAY
+!
+router bfd
+   multihop interval 300 min-rx 300 multiplier 3
+!
+router bgp 65666
+   router-id 172.31.255.23
+   maximum-paths 4 ecmp 4
+   no bgp default ipv4-unicast
+   neighbor EVPN-OVERLAY-PEERS peer group
+   neighbor EVPN-OVERLAY-PEERS next-hop-unchanged
+   neighbor EVPN-OVERLAY-PEERS update-source Loopback0
+   neighbor EVPN-OVERLAY-PEERS bfd
+   neighbor EVPN-OVERLAY-PEERS ebgp-multihop 3
+   neighbor EVPN-OVERLAY-PEERS send-community
+   neighbor EVPN-OVERLAY-PEERS maximum-routes 0
```

Comment:
Hmmmm we should disable EVPN

Reply:
Hmm good point, something like overlay: false. I can't recall the key but I'll find it

Reply:
Done, running CI now